### PR TITLE
Fix the bug where the field expects a string but gets an integer

### PR DESCRIPTION
**Pull request by jiaqiluo**
issue https://github.com/rancher/backup-restore-operator/issues/90

following actions if this PR is merged:
- build a new image
- cut an RC in this repo 
- cut a chart RC in rancher/charts

### CODE REVIEW

**Comment by cbron** [comment]
If this is 2.6 we can't merge or tag until post 2.5.8

**Comment by nickgerace** [comment]
@jiaqiluo is double quoting a potential issue?

**Comment by jiaqiluo** [comment]
> @jiaqiluo is double quoting a potential issue?

@nickgerace Helm will remove the quotes (`"`) around the input when it renders the final template YAML files, so double quoting will never happen.  If the quotes are part of the string, users need to escape it in the input, for example `"\"123123\""`

File: charts/rancher-backup/templates/s3-secret.yaml [hunk 1]
```diff
@@ -14,13 +14,13 @@ stringData:
...
   {{- end }}
   {{- if .region }}
-  region: {{ .region }}
+  region: {{ .region | quote }}
   {{- end }}
-  bucketName: {{ required "A valid .Values.bucketName is required for configuring S3 compatible storage as the default backup storage location" .bucketName }}
+  bucketName: {{ required "A valid .Values.bucketName is required for configuring S3 compatible storage as the default backup storage location" .bucketName | quote }}
   {{- if .folder }}
-  folder: {{ .folder }} 
+  folder: {{ .folder | quote }}
```
Comment (cbron) [inline charts/rancher-backup/templates/s3-secret.yaml:21]: Can you explain why this is needed in the commit or PR description. What string broke it.

Comment (jiaqiluo) [reply charts/rancher-backup/templates/s3-secret.yaml:21]: the commit message is updated.

**Review by ryansann — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**